src/System.Private.ServiceModel/src/System/ServiceModel/Dispatcher/StreamFormatter.cs

```diff
@@ -73,15 +73,15 @@ private Stream GetStreamAndWriteStartWrapperIfNecessary(XmlDictionaryWriter writ
             return streamValue;
         }
 
-        private Task<Stream> GetStreamAndWriteStartWrapperIfNecessaryAsync(XmlDictionaryWriter writer, object[] parameters, object returnValue)
+        private async Task<Stream> GetStreamAndWriteStartWrapperIfNecessaryAsync(XmlDictionaryWriter writer, object[] parameters, object returnValue)
         {
             Stream streamValue = GetStreamValue(parameters, returnValue);
             if (streamValue == null)
                 throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull(_partName);
             if (WrapperName != null)
-                writer.WriteStartElement(null, WrapperName, WrapperNamespace);
-            writer.WriteStartElement(null, PartName, PartNamespace);
-            return Task.FromResult(streamValue);
+                await writer.WriteStartElementAsync(null, WrapperName, WrapperNamespace);
```

Comment:
This is not a tab/spaces issue. The await line above is part of the if statement.